Library/Homebrew/test/rubocops/text_spec.rb

```diff
@@ -87,7 +85,7 @@ def install
       RUBY
     end
 
-    it "When xcodebuild is called without any args" do
+    it "reports an offense if xcodebuild is called without any args" do
```

Comment:
Minor but this could probably be merged into the previous example group (`reports an offense if xcodebuild is called without SYMROOT`)